content/2_Bronze/Intro_Complete.problems.json

```diff
@@ -88,6 +88,18 @@
         "kind": "internal"
       }
     },
+    {
```

Comment:
hmmm why add to the module? iirc it was in the module at some point, then removed? (or maybe i'm remembering wrong) if it was removed, there was probably a good reason

Reply:
History says it was never removed, I thought it was a good fit for complete search. If you disagree though, that's fine

Reply:
oops it used to be in complete search with recursion, not this module (also, history was messed up by problems refactor)
see #1688 for reasoning for why it shouldn't be added to this module

Reply:
I can understand that, but I think it should still be somewhat more accessible. Currently, there's no way to find this problem without looking it up

Reply:
the problem is in the usaco monthlies table https://usaco.guide/general/usaco-monthlies?lang=cpp

maybe you could add a note like (solution here) and link to the internal solution where it's mentioned in the complete-rec module?